src/linters/rubocop.js

```diff
@@ -30,9 +31,14 @@ class RuboCop {
 			throw new Error("Ruby is not installed");
 		}
 
-		// Verify that RuboCop is installed
-		if (!(await commandExists("rubocop"))) {
-			throw new Error(`${this.name} is not installed`);
+		const commandPrefix = prefix("rubocop");
+
+		// if using bundler, commandExists will always fail
+		if (!commandPrefix || !commandPrefix.includes("bundle")) {
+			// Verify that RuboCop is installed
+			if (!(await commandExists("rubocop"))) {
+				throw new Error(`${this.name} is not installed`);
+			}
 		}
```

Comment:
Could you please also add a check if Rubocop is run through Bundler? Similar to how it's done for ESLint:

https://github.com/samuelmeuli/lint-action/blob/23df435b30ce9613184d542280ff2d93579518bb/src/linters/eslint.js#L24-L30

The error code of the version command should be an indicator of the program's presence.

Reply:
Ah - that's a much better way of doing it. Was wondering why `commandExists` wouldn't work right. :)